YogaKit/Source/YGLayout.m

```diff
@@ -126,6 +127,7 @@ - (instancetype)initWithView:(UIView*)view
     YGNodeSetContext(_node, (__bridge void *) view);
     _isEnabled = NO;
     _isIncludedInLayout = YES;
+    _isUIView = [view isMemberOfClass:[UIView class]];
```

Comment:
Here the `view` passed in the initialiser is of type `UIView`. Why do we need `_isUIView` variable?

Reply:
Because it could also be a UILabel, UIButton and any other subclass of UIView. isMemberOfClass makes sure that the parameter is exactly UIView, not a subclass.